server.js

```diff
@@ -99,14 +103,56 @@ function handleWeather(request, response) {
 //   }
 }
 
-
-
-
 function Weather(data) {
   this.time = data.time;
   this.forecast = data.summary;
 }
 
+
+
+
+// SQL locations
+
+app.get('/location', (req,res) => {
+  
+  const SQL = 'SELECT * FROM location';
+  
+  client.query(SQL)
+    .then( results => {
+      if( results.rowCount >= 1 ) {
+        res.status(200).json(location);
+      }
+      else {
+        res.status(400).send('No Results Found');
+      }
+    })
+    .catch(err => res.status(500).send(err));
+});
+  
+app.get('/new', (req,res) => {
+
+  
+  let SQL = `
+      INSERT INTO location (latitude, longitude)
+      VALUES($1, $2)
+    `;
+  
+  let VALUES = [req.query.latitude, req.query.longitude];
+  
+  client.query(SQL, VALUES)
+    .then( results => {
+      if ( results.rowCount >= 1 ) {
+        res.status(301).redirect('https://us1.locationiq.com/v1/search.php');
+      }
+      else {
+        res.status(200).send('Not Added');
+      }
+    })
+    .catch(err => res.status(500).send(err));
+  
+});
```

Comment:
You wouldn't necessarily want a `/new` route.  Perhaps for testing, but not for the live version.  What you should do with this instead is put this logic within your `/location` route.  If the city that you have searched for does not exist within our database, let's find it, save it to the database, and send it to the screen.